### PR TITLE
REL-3453: Made Altair LGS unavailable for 2019A

### PR DESCRIPTION
**Pull request by sraaphorst**
The laser at GN is not working and thus will not be available for 2019A proposals.

This PR:

1. Removes the LGS options from Altair;

2. Issues a warning if upconverting with an observation requiring Altair LGS; and

3. Produces an error (that disallows submission) if the proposal contains an observation with Altair LGS.

Here is a screenshot that shows the Altair node in the resource selection:
screen shot 2018-07-31 at 3 48 54 pm

### CODE REVIEW

File: bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/immutable/transform/UpConverter.scala [hunk 1]
```diff
@@ -206,7 +211,7 @@ case object SemesterConverter2018ATo2018B extends SemesterConverter {
 
   val (texesRemover, texesRemoverMessage) = removeBlueprint("texes", "Texes")
 
-  override val transformers = List(gmosnAltairRemover, dssiGNToAlopeke, phoenixSite, texesRemover)
+  override val transformers = List(dssiGNToAlopeke, phoenixSite, texesRemover)
```
Comment (cquiroz) [inline bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/immutable/transform/UpConverter.scala]: Why is this removed? Because Altair won't be available for any instrument rather that only for gmosN?

Comment (sraaphorst) [reply bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/immutable/transform/UpConverter.scala]: Yes, exactly that. It seems like a pointless check at this point.

I was hemming and hawing about removing it entirely since it only applied to that one semester and probably won't be applicable moving forward.

Comment (cquiroz) [reply bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/immutable/transform/UpConverter.scala]: Fair enough
